docs/advanced-function-deployment.md

```diff
@@ -49,6 +49,21 @@ As any Kubernetes object, function objects have a maximum size of 1.5MiB (due to
   function-content-type: url+zip
 ```
 
+## Functions with bundled deps file
+
+Since the ``requirement.txt``(function deps file) will become long and difficult to put into kubernetes object as function getting complex, Kubeless support use the deps file in remote zip file with function.
+
+Usage: 
+
+- 1.Compress your function and ``requirement.txt``(Dependent on runtime) into a zip file
+- 2.add ``+deps`` into ``function-content-type``
```

Comment:
Note that the name `requirement.txt` filename is only valid for python (for example, for nodejs this is the `package.json`). You can just call this the "dependencies file"

Reply:
Done :)